src/app.js

```diff
@@ -48,6 +52,20 @@ angularAPP.config(function ($routeProvider) {
   // $locationProvider.html5Mode(true);
 });
 
+angularAPP.run(['$route', '$rootScope', '$location', function ($route, $rootScope, $location) {
+    var original = $location.path;
```

Comment:
FYI Run blocks - get executed after the injector is created and are used to kickstart the application. Only instances and constants can be injected into run blocks. This is to prevent further system configuration during application run time.